src/main/java/org/sagebionetworks/web/server/servlet/filter/CORSFilter.java

```diff
@@ -29,93 +29,23 @@ public class CORSFilter extends OncePerRequestFilter {
     "www",
     "staging",
     "tst",
-    "synapse-staging",
-    "status",
-    "staging-signin",
     "signin",
-    "prod",
-    "prod.signin",
-    "nbconvert",
-    "docker",
-    "cdn-www",
-    "cdn-staging",
-    "cdn-tst",
-    "api",
     // Data portals
     "adknowledgeportal",
-    "prod.adknowledgeportal",
-    "staging.adknowledgeportal",
-    "tst.adknowledgeportal",
     "alzdrugtool",
-    "staging.alzdrugtool",
     "arkportal",
-    "prod.arkportal",
-    "staging.arkportal",
     "bsmn",
-    "prod.bsmn",
-    "staging.bsmn",
     "cancercomplexity",
-    "help.cancercomplexity",
-    "prod.cancercomplexity",
-    "staging.cancercomplexity",
     "www.cancercomplexity",
```

Comment:
This was the only portal subdomain I saw that 
1. Uses auth and
2. Doesn't seem to redirect -- keeping it here in case this site is accessed often